LiskHQ/lisk-mobile
# Style issues related to rebranding v2 - Closes #890

# What was the bug or feature?
Described in #890.

## Type of change
- Enhancement (a non-breaking change which adds functionality)

### How to test it?
- Check that transactions other than transfers don't show the amount on the transactions page.
- Make sure tab bar border is in line with design.


# Checklist:
- My code follows the style guidelines of this project
- I have performed a self-review of my own code
- I have commented my code, particularly in hard-to-understand areas
- I have made corresponding changes to the documentation
- I have added tests that prove my fix is effective or that my feature works
- New and existing unit tests pass locally with my changes
